go.mod

```
@@ -4,6 +4,6 @@ go 1.13
 
 require (
 	github.com/elastic/cloud-sdk-go v1.0.0
-	github.com/hashicorp/terraform-plugin-sdk v1.15.0
+	github.com/hashicorp/terraform-plugin-sdk/v2/v2 v2.0.0
```

I'll take a look at this module update, I believe it added an extra `/v2` to the module. It should be:
```suggestion
	github.com/hashicorp/terraform-plugin-sdk/v2 v2.0.0
```